src/Discord.Net.Rest/Extensions/ClientExtensions.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using System.Threading.Tasks;
+
+namespace Discord.Rest.Extensions
```

Comment:
```suggestion
namespace Discord.Rest
```
Namespace is inconsistent with the other extension class in this directory.